.github/labeler.yml

```
@@ -0,0 +1,2 @@
+shared-web:
+- webui/react/src/shared/**/*
```

```suggestion
- webui/react/src/shared/**
```
You only need the trailing `/*` if you want something like `*.js` in any directory. If it's "all the files in any subdirs", then /** matches.

I'm pretty sure it'll still work the way it is, but if the Readme spelling is fixed, may as well clean this up at the same time. :)